easybuild/easyconfigs/v/Voro++/Voro++-0.4.6-foss-2020a.eb

```diff
@@ -0,0 +1,39 @@
+
+easyblock = 'ConfigureMake'
+
+name = 'Voro++'
+version = '0.4.6'
+
+homepage = 'http://math.lbl.gov/voro++/'
+description = """Voro++ is a software library for carrying out three-dimensional computations of the Voronoi
+tessellation. A distinguishing feature of the Voro++ library is that it carries out cell-based calculations,
+computing the Voronoi cell for each particle individually. It is particularly well-suited for applications that
+rely on cell-based statistics, where features of Voronoi cells (eg. volume, centroid, number of faces) can be used
+to analyze a system of particles."""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://math.lbl.gov/voro++/download/dir/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['ef7970071ee2ce3800daa8723649ca069dc4c71cc25f0f7d22552387f3ea437e']
+
+# Let's store the checksum in order to be sure it doesn't suddenly change
```

Comment:
Not sure why this comment is here....

Reply:
that seems to come from the time where checksum were not a regular thing :D